lib/src/builders/expression.dart

```diff
@@ -210,8 +240,13 @@ abstract class AbstractExpressionMixin implements ExpressionBuilder {
   }
 
   @override
+  ExpressionBuilder increment([bool prefix = false]) {
+    return new _IncrementExpression(this, prefix);
+  }
+
+  @override
   ExpressionBuilder identical(ExpressionBuilder other) {
-    return lib$core.identical.call([
+    return lib$core.identical.call(<ExpressionBuilder>[
```

Comment:
is this fixing an analyzer warning? I'd be surprised if this wasn't inferred in strong mode...